frontend/dashboard/src/styles.css

```diff
@@ -0,0 +1,114 @@
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #f0f0f0;
+  }
+
+  .dashboard {
+    display: flex;
+    height: 100vh;
+  }
+
+  .sidebar {
+    width: 250px;
+    background-color: #3b3f54;
+    color: white;
+    padding: 20px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .sidebar h1 {
+    font-size: 1.5em;
+    margin-bottom: 30px;
+  }
+
+  .sidebar nav ul {
+    list-style: none;
+    padding: 0;
+    flex-grow: 1;
+  }
+
+  .sidebar nav ul li {
+    margin: 20px 0;
```

Comment:
I see lots of px.... But I feel like ATC might have mercy on us, this is super tough right now. Just try not to use it going forward.

Reply:
Noted going forward, and if there's time I'll replace the PXs